pytorch_lightning/trainer/trainer.py

```diff
@@ -543,8 +538,8 @@ def validate(
                 or a :class:`~pytorch_lightning.core.datamodule.LightningDataModule` specifying validation samples.
 
             ckpt_path: Either ``best`` or path to the checkpoint you wish to validate.
-                If ``None``, use the current weights of the model.
-                When the model is given as argument, this parameter will not apply.
+                If ``None``, use the current weights of the model if provided, or the best model from ``trainer.fit``.
+                When the model and the ckpt path are passed as arguments, we load the ckpt path.
```

Comment:
Rewording and removing the last part. Needs a line break and updating the docstring of the other methods

```suggestion
                If ``None`` and the model instance was passed, use the current weights. Otherwise, the best model from the previous ``trainer.fit`` call will be loaded.
```